### DISCUSSION TITLE
Feature Request: Option to Modify Keybind Helper in gum choose

### DISCUSSION THREAD

**Discussion by fykaa** [charmbracelet/gum#623] [Q&A]
I am currently using gum choose to create a selection menu for various project languages. The command works well, but I would like to simplify the displayed keybind helpers by removing the filter keybind helper from the UI.

Current Command:

```
gum choose --height 8 --cursor "• " --cursor-prefix "[•]" --unselected-prefix "[ ]" --selected-prefix "[✔]" --header "" --header.foreground 81 "Python" "Ruby" "Go" "NuGet"
```

Issue:
The keybind helpers currently show the filter keybind, which is not needed for my use case. 
![image](https://github.com/user-attachments/assets/74dbd25c-d87b-4f9f-aaf2-cee09728f907)

Is there a way to customise keybinds for gum commands in shell scripts

**Comment by wellkilo** [comment]
Yes — you can hide the entire keybind help footer for `gum choose`:

```bash
gum choose --no-show-help \
  --height 8 \
  --cursor '• ' \
  --header '' \
  Python Ruby Go NuGet
```

The equivalent environment variable is useful for a whole script:

```bash
export GUM_CHOOSE_SHOW_HELP=false
```

Current Gum exposes `ShowHelp` as a negatable option with a default of `true`, so Kong generates `--no-show-help`; the environment key is `GUM_CHOOSE_SHOW_HELP` ([source](https://github.com/charmbracelet/gum/blob/7179388031ae67d7f538d001be87d931f1cf5e28/choose/options.go#L12-L31)).

This hides the complete helper line. There is not currently a CLI option for selectively hiding only the filter binding while retaining the other help bindings; that would require a more granular keymap/help configuration feature.

Also note that for the default single-selection mode, selected/unselected prefixes are intentionally hidden and only the cursor is shown. If this gives you the simplified UI you wanted, please mark the reply as accepted.
